src/status_im/protocol/group.cljs

```diff
@@ -136,9 +137,11 @@
     (fn [key-id]
       (f/add-filter!
         web3
-        {:topics [f/status-topic]
-         :key    key-id
-         :type   :sym}
+        (cond-> {:topics   [f/status-topic]
```

Comment:
Is there a specific reason we are using cond-> here? With flags I think it's generally more clear / less error prone if it goes like this:

```
(if config/offline-inbox-enabled?
   experimental-behavior
   old-behavior)
```

In this case, it seems like `allowP2P true` slipped into the old behavior.

Reply:
@oskarth i actually thought about `if` vs `cond->` here and decided to go with latter. Agree that `if` looks better if we actually want to remove branching in future. Will change that.